android/app/src/main/kotlin/org/getlantern/lantern/util/LanternHttpClient.kt

```diff
@@ -65,8 +65,10 @@ open class LanternHttpClient : HttpClient() {
 
             override fun onSuccess(response: Response?, result: JsonObject?) {
                 result?.let {
-                    val user: ProUser = JsonUtil.fromJson<ProUser>(result.toString())
-                    cb.onSuccess(response!!, user)
+                    val user: ProUser? = JsonUtil.tryParseJson<ProUser?>(result.toString())
```

Comment:
@atavism The only issue is here, while parsing JSON, if we get into any issue,  I guess it will never return a response and even if it returns a response then it will return a null guess.

With an older approach, if we get into parsing issue, It will still parse some basic values and return results, so app does not crash.

Reply:
My thinking with this change is if the API suddenly behaves in some unexpected manner (returns null for some field when it previously always returned an empty array), we should treat that response as invalid and skip over it vs. trying to extract valid fields from it. With either approach, the app shouldn't crash now